evaluation.md

```diff
@@ -1,19 +1,68 @@
 # Evaluation: running commentary
 
-## Internal DSL
+Please see [reflection.md](/reflection.md) for additional comments on our work.
 
-_Describe each change from your ideal syntax to the syntax you implemented, and
-describe_ why _you made the change._
+## Internal DSL
 
 **On a scale of 1–10 (where 10 is "a lot"), how much did you have to change your syntax?**
 
+3. Our ideal syntax isn't too different from the internal DSL we wrote. The main 
+features such as going to a certain surrounding, jumping from a label to the other, 
+and facing a specific direction remain the same. There are only two main differences.
+Firstly, we were not able to write an `until` method so that instead of saying
+`Go(<surrounding>)` we could say `Go until <surrounding>`. Secondly, to translate the internal
+AST into the provided API we realized we needed to have our `Label` take in a list of rules
+and that we needed a wrapper case class that took in a list of `Label`s. This meant we
+had to have a program structure like:
```

Comment:
There are ways to not need the list, but it requires a lot of nonsense syntax that exists just to give you alternating method/argument phrases